packages/components/psammead-most-read/src/index.jsx

```diff
@@ -41,20 +42,31 @@ MostReadSection.propTypes = {
   dir: oneOf(['rtl', 'ltr']),
   service: string.isRequired,
   header: string.isRequired,
+  className: string,
 };
 
 MostReadSection.defaultProps = {
   labelId: 'Most-Read',
   dir: 'ltr',
+  className: null,
 };
 
-const MostRead = ({ items, script, service, header, dir, labelId }) => (
+const MostRead = ({
+  items,
+  script,
+  service,
+  header,
+  dir,
+  labelId,
+  className,
+}) => (
   <MostReadSection
     labelId={labelId}
     script={script}
     service={service}
     header={header}
     dir={dir}
+    className={className}
```

Comment:
Can you give me an example of how this `className` is going to be used? 
The README should have been updated with this new `prop`.
@mmcculloch @lokh01